app/containers/Login/index.js

```diff
@@ -0,0 +1,163 @@
+/**
+ *
+ * Login
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+
+import injectReducer from 'utils/injectReducer';
+
+import UserIsAbsentInSystem from 'components/UserIsAbsentInSystem';
+import SelectAccountComponent from 'components/SelectAccount';
+import ScatterInstaller from 'components/ScatterInstaller';
+import ModalDialog from 'containers/ModalDialog';
+
+import {
+  NO_SCATTER,
+  NO_SELECTED_SCATTER_ACCOUNTS,
+  USER_IS_ABSENT_IN_SYSTEM_AND_LOGIN,
+} from 'containers/SignUp/constants';
+
+import {
+  reloadApp,
+  loginSignup,
+  forgetIdentity,
+} from 'containers/AccountInitializer/actions';
+
+import { showSignUpModal } from 'containers/SignUp/actions';
+import { makeSelectUserIsInSystem } from 'containers/AccountInitializer/selectors';
+
+import LoginPopup from './LoginPopup';
+
+import {
+  LOGIN_MODAL_ID,
+  SHOW_DEFAULT_LOGIN_MODAL,
+  COMPLETE_LOGIN,
+} from './constants';
+
+import { showLoginModal, hideLoginModal } from './actions';
+import { makeSelectContent, makeSelectShowModal } from './selectors';
+import reducer from './reducer';
+
+/* eslint-disable react/prefer-stateless-function */
+export class Login extends React.Component {
+  componentDidUpdate() {
+    const { userIsInSystem, showLoginModalDispatch } = this.props;
+    const reload = localStorage.getItem(COMPLETE_LOGIN);
+    const scrollTo = localStorage.getItem('scrollTo');
+
+    if (reload && userIsInSystem !== null) {
+      showLoginModalDispatch();
+      if (scrollTo) window.scrollTo(0, +scrollTo);
+      localStorage.clear();
+    }
+  }
+
+  continueLogin = () => {
+    this.props.loginSignupDispatch({ type: COMPLETE_LOGIN });
+  };
+
+  backToOptions = () => {
+    this.props.showLoginModalDispatch(SHOW_DEFAULT_LOGIN_MODAL);
+  };
+
+  openSignUpWindow = () => {
+    this.props.hideLoginModalDispatch();
+    this.props.showSignUpModalDispatch();
+  };
+
+  render() {
+    const {
+      content,
+      forgetIdentityDispatch,
+      reloadAppDispatch,
+      showModal,
+      hideLoginModalDispatch,
+    } = this.props;
+
+    return (
+      <ModalDialog
+        modalId={LOGIN_MODAL_ID}
```

Comment:
Is it possible to get rid of that prop? If need to use ModalDialog component and modalId does not provide any functionality, I do not want to worry about setting the value for it.